RelayTestApp/Assets/Scripts/brainCloud/BrainCloudManager.cs

```diff
@@ -0,0 +1,443 @@
+using System.Collections;
```

Comment:
Is this file part of brainCloud? Why is it under the brainCloud/ folder. This should probably be called "App" or "Game" or something. Anything under brainCloud/ should be left untouched, it comes from the plugin.